.github/workflows/tests.yml

```diff
@@ -19,7 +19,7 @@ permissions:
   contents: read
 
 env:
-  cache-version: v2
+  cache-version: v3
```

Comment:
Is this actually needed?

Reply:
Not necessary. I just want it to re-generate cache files for CI especially for instrumentation tests.